TipCatDotNet.Api/Services/Payments/PayoutService.cs

```diff
@@ -0,0 +1,94 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using Stripe;
+using TipCatDotNet.Api.Data;
+
+namespace TipCatDotNet.Api.Services.Payments;
+
+public class PayoutService : IPayoutService
+{
+    public PayoutService(AetherDbContext context, Stripe.PayoutService payoutService)
+    {
+        _context = context;
+        _payoutService = payoutService;
+    }
+
+
+    public Task<Result> PayoutAll(CancellationToken cancellationToken = default)
+    {
+        return Result.Success()
+            .Bind(PayoutByUsers);
+
+
+        async Task<Result> PayoutByUsers()
+        {
+            Result result = Result.Success();
+            var allStripeAccounts = await _context.StripeAccounts
+                .Join(_context.Members.Where(m => m.IsActive), s => s.MemberId, m => m.Id, (s, m) => s.StripeId)
+                .ToListAsync(cancellationToken);
+
+            allStripeAccounts.ForEach(async stripeAccointId =>
+            {
+                var (_, IsFailure, error) = await Payout(stripeAccointId);
+                if (IsFailure)
```

Comment:
Not sure if we need fail in just one payment fails. We may delay it execution to the next run. Better only log a result